ADAPTERS.md

```diff
@@ -1,28 +1,40 @@
 
 In our opinion the easiest way to create an adapter package is as follows:
 
 1. Fork this repository
 1. Create a folder in the [packages](./packages) folder for your adapter
-1. Add a test institution to the [default institution list](./apps/server/cachedDefaults/ucwInstitutionsMapping.json) with aggregator support defined for your institution.
+1. Add a test institution to the [default institution list](./apps/server/cachedDefaults/ucwInstitutionsMapping.json) with aggregator support defined for your institution
 1. Build your adapter
 1. Import your adapter in [adapterSetup.ts](./apps/server/src/adapterSetup.ts) on your forked version of this repo
 1. Write unit, integration, and e2e tests
 1. Bundle and publish your adapter to npm
-1. When your adapter package is ready for production use, then you'll need to gain access to the UCP institution list to update the list of institutions you support(This functionality is in progress by the UCP team)
+1. When your adapter package is ready for production use, then you'll need to gain access to the UCP institution list to update the list of institutions you support (This functionality is in progress by the UCP team)
+
+We are open to contributions to make this process easier.
+
+An example adapter lives [here](./apps/server/src/test-adapter/index.ts). Each adapter needs to export an adapterMapObject. These adapterMapObjects are imported and setup by someone hosting the UCW in [adapterSetup.ts](./apps/server/src/adapterSetup.ts). If the test adapterMapObject doesn't support a property you need to use for your adapter, then you'll need to add it to the test example adapter to ensure it continues to get support.
+
+This repo accesses adapter-specific logic in [adapterIndex.ts](./apps/server/src/adapterIndex.ts)
 
-An example adapter lives [here](./apps/server/src/test-adapter/index.ts). Each adapter needs to export an adapterMapObject. These adapterMapObjects are imported and setup by hosters of the UCW in [adapterSetup.ts](./apps/server/src/adapterSetup.ts). If the test adapterMapObject doesn't support a property you need to use for your adapter, then you'll need to add it to the test example adapter to ensure it continues to get support.
+## Monorepo 
 
-This repo accesses adapter specific logic via [adapterIndex.ts](./apps/server/src/adapterIndex.ts)
+Because this repo is a monorepo, there are some caveats to consider with regard to adapter creation. See [the MONOREPO readme](MONOREPO.md#creating-an-aggregator-adapter-package-within-a-monorepo) for more details.
```

Comment:
This feels like it's making the Monorepo page much more complicated. Since creating adapters is so rarely done, I think it might make sense to keep everything related to it all together, rather than introducing complexity for a fintech engineer (I'm thinking through the different users who will be using this, and this is for the aggregator engineer, and the other page is more geared toward the fintech engineer).

The items on the Monorepo page seem almost like sub-steps under "Build your adapter" within this page. I'm thinking it would make sense to add more detail to each of the steps here. So instead of "Build your adapter", put the detail from the Monorepo packaging. "Add a test institution" could have an example of what the test institution would look like.

This might be a bigger ask, so we could also create a separate ticket for it.

Reply:
Great feedback. So, keep the monorepo readme unchanged, and just add more details to the Adapters readme?

Reply:
Exactly :+1:

Reply:
Okay, @emilysallstrom I made some changes, including adding info about test institutions. Give it a look when you have a minute. Thank you!